### ISSUE TITLE
Reconfigure landmarks on search results page

### ISSUE THREAD

**Issue by lhenze** [nyudlts/ultraviolet#135]
From NYU A11y Review: 

Landmarks should be reconfigured on Search Results page

The Search results page has a banner area, a navigation area, and then should move into the main area to encompass the Filters section and the results section. The left-hand side search filter/form could be contained in a 'complementary' section with an aria-label="filters".
